ios/Classes/MapboxMapController.swift

```diff
@@ -332,7 +332,16 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
                 }
             }
             result(nil)
-
+        case "map#setPadding":
+            guard let arguments = methodCall.arguments as? [String: Any] else { return }
+            guard let left = arguments["left"] as? Float else { return }
+            guard let right = arguments["right"] as? Float else { return }
+            guard let top = arguments["top"] as? Float else { return }
+            guard let bottom = arguments["bottom"] as? Float else { return }
+            
+            
+            mapView.contentInset = UIEdgeInsets(top: CGFloat(top), left: CGFloat(left), bottom: CGFloat(bottom), right: CGFloat(right))
+            result(nil)
```

Comment:
Once you did the open todos you will eventually adjust this line of code and call this method with the provided animation duration. Keep in mind 2 things:
- most iOS apis treat animation durations as seconds (where others use ms)
- Add a completion handler to the call and only invoke `result(nil)` once the animation is done